drivers/lc709203f/include/lc709203f_params.h

```diff
@@ -0,0 +1,38 @@
+#ifndef LC709203F_PARAMS_H
+#define LC709203F_PARAMS_H
+
+#include "board.h"  /* THIS INCLUDE IS MANDATORY */
+#include "lc709203f.h"
+
+/**
+ * @brief   Default configuration parameters for LC709203F sensors
+ * @{
+ */
+#ifndef LC709203F_PARAMS_I2C
+#define LC709203F_PARAMS_I2C            (I2C_DEV(0))
+#endif
+#ifndef LC709203F_PARAMS_ADDR
+#define LC709203F_PARAMS_ADDR           (0xb)
+#endif
+#ifndef LC709203F_PARAMS_ALARM_PIN
+#define LC709203F_PARAMS_ALARM_PIN      GPIO_PIN(PORT_E, 6)
```

Comment:
It's probably better to change this to GPIO_PIN(4,  6) to allow compilation for boards that don't have PORT_XY definitions.